snowpack/src/commands/build.ts

```diff
@@ -289,6 +296,10 @@ export async function command(commandOptions: CommandOptions) {
 
   // Write the `import.meta.env` contents file to disk
   await fs.writeFile(path.join(internalFilesBuildLoc, 'env.js'), generateEnvModule('production'));
+  if (getUseHmr(config)) {
+    await fs.copyFile(require.resolve('../../assets/hmr.js'), path.resolve(internalFilesBuildLoc, 'hmr.js'));
```

Comment:
in `dev.ts` we do:

```
const HMR_DEV_CODE = readFileSync(path.join(__dirname, '../assets/hmr.js'));
```

can you move this constant into our `util` file, and then use it directly here? I'd like to get as much code sharing with the dev HMR as possible.

Reply:
add the `hmrFilePath` in [util file](https://github.com/pikapkg/snowpack/pull/1008/files#diff-38ad01639604030e31a8749cc30c8c9dR413)